innopoints/config/prod.py

```diff
@@ -7,13 +7,15 @@
 INNOPOLIS_SSO_CLIENT_ID = os.environ['INNOPOLIS_SSO_CLIENT_ID']
 INNOPOLIS_SSO_CLIENT_SECRET = os.environ['INNOPOLIS_SSO_CLIENT_SECRET']
 
-CORS_ORIGINS = 'https://innopoints-frontend.herokuapp.com'
+CORS_ORIGINS = ['https://innopoints-frontend.herokuapp.com']
```

Comment:
Why do even need CORS in production? We have a config file for Heroku, so this should be for the final server hosted on Innopolis servers, where they would be on the same origin.

Reply:
Well, as of know, we know nothing about the production environment, and, really, the production config is mostly random, so just stick with it